#!/usr/bin/env python
PACKAGE = "my_package"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("N_scan", int_t, 0, "An Integer parameter", 0, 1738, 5000)
gen.add("err_x", double_t, 0, "A Double parameter", 5.23428, -500, 500) # -71.5 for stk map
gen.add("err_y", double_t, 0, "A Double parameter", 1.16272, -500, 500) # 23.8 ~
gen.add("err_yaw", double_t, 0, "A Double parameter", 0.00717,	-3.142, 3.142) # 0.223 ~

exit(gen.generate(PACKAGE, "my_package", "offsetMapping"))
